src/types/box.js

```diff
@@ -141,6 +141,9 @@ module.exports = function(Chart) {
 			ctx.strokeRect(view.left, view.top, width, height);
 
 			ctx.restore();
+		},
+		drawLabel: function() {
```

Comment:
Maybe cleaner to instead check `if (element.drawLabel)` in `annotation.js` instead?

Reply:
Labels for boxes is [on the roadmap](https://github.com/chartjs/chartjs-plugin-annotation#to-do-items), so this frames it out for that.